servers/plugins/org.fusesource.ide.server.karaf.core/src/org/fusesource/ide/server/karaf/core/server/subsystems/Karaf2xPublishController.java

```diff
@@ -233,8 +240,23 @@ public int removeModule(IModule[] module, IProgressMonitor monitor) throws CoreE
 
 	private IPath getTempBundlePath(IModule[] module) {
 		IPath localTempLocation = getMetadataTemporaryLocation(getServer());
-		String archiveName = module[0].getName() + ServerModelUtilities.getDefaultSuffixForModule(module[0]);
+		String moduleVersion = null;
+		try {
```

Comment:
Indention seems to be crappy. Please check if your eclipse uses tabs and not spaces.